automatic/keeweb/update.ps1

```diff
@@ -34,7 +34,7 @@ function global:au_GetLatest {
     $url32SegmentSize = $([System.Uri]$url32).Segments.Length
     $filename32 = $([System.Uri]$url32).Segments[$url32SegmentSize - 1]
 
-    $url64 = $downloadedPage.links | ? href -match '64.exe$' | select -First 1 -expand href
```

Comment:
The 64.exe was selecting the arm64.exe version not the x64.exe
So i've added the x to filter on it too